src/core/radio/radio.hpp

```diff
@@ -45,7 +45,7 @@
 
 namespace ot {
 
-static constexpr uint32_t kUsPerTenSymbols = OT_US_PER_TEN_SYMBOLS; ///< The microseconds per 10 symbols.
+static constexpr uint32_t kUsPerTenSymbols = OT_RADIO_TEN_SYMBOLS_TIME_US; ///< The microseconds per 10 symbols.
```

Comment:
Change the internal name `kUsPerTenSymbols`?

Reply:
Let's keep this unchanged since we will keep `OT_US_PER_TEN_SYMBOLS`.